repos/core-setup.proj

```diff
@@ -9,25 +9,30 @@
     <OverridePortable>$(PortableBuild)</OverridePortable>
     <OverridePortable Condition="'$(TargetOS)' == 'OSX'">true</OverridePortable>
 
-    <BuildArguments>-ConfigurationGroup=$(Configuration)</BuildArguments>
-    <BuildArguments>$(BuildArguments) -PortableBuild=$(OverridePortable)</BuildArguments>
-    <BuildArguments>$(BuildArguments) -SkipTests=true</BuildArguments>
-    <BuildArguments Condition="$(Platform.Contains('arm'))">$(BuildArguments) -TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true</BuildArguments>
-    <BuildArguments>$(BuildArguments) --</BuildArguments>
+    <BuildArguments>--restore --build</BuildArguments>
+    <BuildArguments>$(BuildArguments) --configuration $(Configuration)</BuildArguments>
+    <BuildArguments>$(BuildArguments) --ci</BuildArguments>
```

Comment:
Although it probably wouldn't matter, but could FlagParameterPrefix replace '--', I am not sure though.

Reply:
Yes, fixed.